content/docs/reference/crosswalk/aws/ecs.md

```diff
@@ -57,20 +57,21 @@ const nginx = new awsx.ecs.FargateService("nginx", {
 export const url = lb.endpoint.hostname;
 ```
 
-After deploying this program, we can access our 2 Nginx web servers behind our load balancer:
+After deploying this program, we can access our two  NGINX web servers behind our load balancer:
 
 ```bash
-$ curl https://$(pulumi stack output url)
+$ curl http://$(pulumi stack output)
 <!DOCTYPE html>
 <html>
 <body>
 <h1>Welcome to nginx!</h1>
 </body>
 </html>
 ```
+Replace `$(pulumi stack output)` with the resulting URL value of `pulumi stack output`.
```

Comment:
I wasn't sure about this. The idea of using `pulumi stack output url`, with the `url` part in there, is that you don't need to do any manual substitution. That should evaluate to the load balancer's URL.